src/instructlab/configuration.py

```diff
@@ -540,6 +540,32 @@ class _train(BaseModel):
     )
 
 
+class _metadata(BaseModel):
+    # model configuration
+    model_config = ConfigDict(
+        extra="ignore",
+        protected_namespaces=(),
```

Comment:
Do we need to use protected_namespaces()?

Reply:
This is a standard for new classes we introduce